app/views/article-amp.html

```diff
@@ -447,6 +447,8 @@ <h1 class="related">Related articles</h1>
         <a subscriptions-action="subscribe" subscriptions-display="TRUE">Subscribe</a>
         or
         <a subscriptions-action="login" subscriptions-display="TRUE">Login</a>
+        <div subscriptions-action="subscribe" subscriptions-display="TRUE"
+          subscriptions-service="subscribe.google.com" subscriptions-decorate>Subscribe</div>
```

Comment:
`subscription-decorate` is default, right? Actually, we need to remove it in either case from here for now - Scenic must always be valid, so we can't use `decorate` until validator is updated everywhere.